frontend/src/static/js/components/webstatus-columns-dialog.ts

```diff
@@ -58,41 +68,70 @@ export class WebstatusColumnsDialog extends LitElement {
 
   handleSave() {
     const newColumns: string[] = [];
+    const columnOptions: string[] = [];
     this.shadowRoot!.querySelectorAll('sl-checkbox').forEach(cb => {
+      if (cb.classList.contains('column-option') && !cb.checked) {
```

Comment:
This is pushing the column option when it is not checked.

Reply:
Hmm, not sure how that happened.  Maybe it was !!cb.checked.